ballerina/response_generator.bal

```diff
@@ -97,6 +97,21 @@ class ResponseGenerator {
         return errorDetail;
     }
 
+    isolated function addErrors(error[] errors, parser:FieldNode fieldNode) {
```

Comment:
Since this method is used only for constraint validations, shall we make the method name specific? `addConstrainValidationErrors`, maybe?